src/index.html

```diff
@@ -0,0 +1,133 @@
+                alt="">
+        </div>
+        <div id="search-area">
+            <form>
+                <input type="text" placeholder="Search">
+                <button id="search"><i class="fa fa-search"></i></button>
+            </form>
+            <button id="mic"><i class="fa fa-microphone" title="Search with your voice"></i></button>
+        </div>
+
+        <div id="right">
```

Comment:
- [ ] You may need to use `class selector` instead of `id selector`.
This is because class selectors give you a chance to identify more than one HTML element.
On addition, i have noticed that you used `class` in `divs`. Kindly consider this to ensure the project is more professional and enables you to make multiple selections